doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -228,7 +228,9 @@ nRF9160 samples
 Peripheral samples
 ------------------
 
-|no_changes_yet_note|
+* :ref:`radio_test` sample:
+
+  * Added a workaround for the hardware Errata 254 of the nRF5340 chip.
```

Comment:
Would you like to add a link to the Errata? Which, by the way, is not for that chip. If you mean this one https://infocenter.nordicsemi.com/topic/errata_nRF52840_Rev3/ERR/nRF52840/Rev3/latest/config_840_254.html.

Reply:
In previous changelogs there were no links, so I don't know if it's ok to add one.
I will correct chip names.

Reply:
It would be ok to add one. So, add one in the `links.txt` file somewhere in the end of the **Source: infocenter.nordicsemi.com** section and use it in the changelog entry.